src/client/components/AreaSelector/CountryList/countryList.scss

```diff
@@ -51,15 +51,16 @@
   display: grid;
   line-height: $spacing-xs;
   padding: $spacing-xxs $spacing-xs;
+  align-items: center;
```

Comment:
This would be with aligncenter:
![image](https://github.com/openforis/fra-platform/assets/41337901/c16b55f0-d581-4713-b787-8343a1e7ac4a)

And this would be without:
![image](https://github.com/openforis/fra-platform/assets/41337901/c616f4b5-94a6-4751-8ebd-7e64f299983e)


The country name wraps too (unrelated to this PR), and this is how it looks:
With align:
![image](https://github.com/openforis/fra-platform/assets/41337901/07f2b1f0-43c8-4d85-855e-0f3ca6368fb2)

Without:
![image](https://github.com/openforis/fra-platform/assets/41337901/30f305bb-c896-49f6-88fc-9b643b32c0a5)


Let me know what you think